### ISSUE TITLE
ALVR v20.11.0 Black Screen on Ubuntu 24.04 - No Display on Quest or ALVR Monitor

### ISSUE THREAD

**Issue by zuhaitz-arbola** [alvr-org/ALVR#2404]
**Description**

I am experiencing a black screen while using ALVR (v20.11.0) with SteamVR on my Oculus Quest. Both the headset and PC seem to connect correctly, but no video is displayed. The ALVR monitor also shows nothing, and the following errors appeared:

"H264 profile does not contain encoding entrypoint"
"HEVC profile does not contain encoding entrypoint"

I have already tried setting WAYLAND_DISPLAY='' %command% and __NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only __GLX_VENDOR_LIBRARY_NAME=nvidia %command% in the SteamVR launch options, but the issue persists. Based on the output of nvidia-smi, it is confirmed that NVIDIA RTX 3060 is being used, and not the integrated GPU.
General Troubleshooting

I carefully followed the instructions in the README and successfully completed the setup wizard.
I read the ALVR GitHub Wiki.

**Environment**

**Hardware**

CPU: Intel Core i7-10750H
GPU: NVIDIA RTX 3060
GPU Driver Version: 550.107.02
Audio: PulseAudio (or PipeWire, if applicable)

**Installation**

ALVR Version: v20.11.0
ALVR Settings File: Available upon request
SteamVR Version: Latest available version from the Steam store
Install Type: Portable (tar.gz)

OS Name and Version: Ubuntu 24.04

**Comment by Macjja** [comment]
Hi there @zuhaitz-arbola , did you look at the [linux troubleshooting guide?](https://github.com/alvr-org/ALVR/wiki/Linux-Troubleshooting) To get it working on linux, we have to add `~/.local/share/Steam/steamapps/common/SteamVR/bin/vrmonitor.sh %command% ` to your command line arguments for SteamVR to work properly, which is listed as the first issue on the guide.

**Comment by zuhaitz-arbola** [comment]
> Hi there @zuhaitz-arbola , did you look at the [linux troubleshooting guide?](https://github.com/alvr-org/ALVR/wiki/Linux-Troubleshooting) To get it working on linux, we have to add `~/.local/share/Steam/steamapps/common/SteamVR/bin/vrmonitor.sh %command% ` to your command line arguments for SteamVR to work properly, which is listed as the first issue on the guide.

Yes I did, I forgot, sorry, I did and then I obtain these messages:

![2024-09-15 23-33-47](https://github.com/user-attachments/assets/27f3d955-960e-49c7-b7de-e07e5fd0f4d3)
![2024-09-15 23-33-38](https://github.com/user-attachments/assets/5a6d05e0-f63c-4f93-93a3-2e61e076afee)

**Comment by snagar** [comment]
I think I have similar issue, or something along those lines.
I'm using Mint OS which is based on Ubuntu too. The WM I'm using is Cinnamon.
What I found is that the NV drivers I have are similar to @zuhaitz-arbola: "GPU Driver Version: 550.107.02" with CUDA Driver 12.4.

I receive the error: 
```
Headset Error
make sure your headset display is in fullscreen mode
Enable fullscreen Mode  -202
```

In my Quest 3, I see blank screen.
I tried different command line options, in "SteamVR", but I got the same outcome.
Interestingly, I can see that my Headset is being recognised and the SteamVR window does open.

I wonder if we have to use Wayland for this to work since on my Nubara 40, all works as expected.

**Comment by jurosic** [comment]
Same here, im getting no errors in ALVR, only in steamvr that my headset is not in fullscreen mode

**Comment by guysoft** [comment]
getting here too, ubuntu 24.10 ALVR v20.11.1, dual GPU setup.
Running directly from the cli
```
~/.local/share/Steam/steamapps/common/SteamVR/bin/vrmonitor.sh %command% 
```

**Comment by ChristOurLife** [comment]
Same issue here. Using Linux Mint with proprietary NVIDIA GT 1030 drivers. This card (DDR5) supports H264.

**Comment by ChristOurLife** [comment]
Same issue here. Using Linux Mint with proprietary NVIDIA GT 1030 drivers. This card (DDR5) supports H264.

**Comment by Sloth-on-meth** [comment]
same issue here

**Comment by guysoft** [comment]
Helped me to fix it:

1. Verify files integrity found something for me:
https://help.steampowered.com/en/faqs/view/0C48-FCBD-DA71-93EB

2. Use ONLY:
```
~/.local/share/Steam/steamapps/common/SteamVR/bin/vrmonitor.sh %command% 
```
adding these made it NOT work:
```
__NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only __GLX_VENDOR_LIBRARY_NAME=nvidia
```

3. I upgraded from Ubuntu 24.10 to 25.04.

I have an Asus G14  model 2020 with nvidia gtx 2060 and amd integrated motherboard
